<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Stylizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_left_panel" native="true">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_load">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Load OSM File</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_load_settings">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Load Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_save">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Save Image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_reset">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Reset Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_general_settings">
         <property name="title">
          <string>General Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_gen_1" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Max Dimension</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_max">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>40000.000000000000000</double>
               </property>
               <property name="value">
                <double>6000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_gen_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_gen_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_bg_color">
               <property name="text">
                <string>Background Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_bg_color">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_line_settings">
         <property name="title">
          <string>Line Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeWidget" name="treeWidget_line_settings">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_line">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>&lt;value&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_line">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_line">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_line">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Solid Line</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dash Line</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dot Line</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dash-Dot Line</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dash-Dot-Dot Line</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_cap">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Square Cap</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Flat Cap</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Round Cap</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_join">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Bevel Join</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Miter Join</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Round Join</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_fill_settings">
         <property name="title">
          <string>Fill Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="treeWidget_fill_settings">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_fill">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>&lt;value&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_fill">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MapWidget" name="widget_map" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapWidget</class>
   <extends>QWidget</extends>
   <header>mapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_bg_color</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bgColorBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadOSMFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_line_settings</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>lineSelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_fill_settings</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>fillSelChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_line</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>lineColorBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_fill</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fillColorBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>591</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_line</sender>
   <signal>valueChanged(double)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineWidthChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_line</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>lineComboChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cap</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>capComboChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_join</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>joinComboChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_line_settings</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkboxChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget_fill_settings</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkboxChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load_settings</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>radioChanged()</slot>
  <slot>bgColorBtnClicked()</slot>
  <slot>loadOSMFile()</slot>
  <slot>saveImage()</slot>
  <slot>lineSelChanged()</slot>
  <slot>fillSelChanged()</slot>
  <slot>lineColorBtnClicked()</slot>
  <slot>fillColorBtnClicked()</slot>
  <slot>lineWidthChanged(double)</slot>
  <slot>lineComboChanged(QString)</slot>
  <slot>capComboChanged(QString)</slot>
  <slot>joinComboChanged(QString)</slot>
  <slot>checkboxChanged(QTreeWidgetItem*,int)</slot>
  <slot>resetBtnClicked()</slot>
  <slot>loadSettingsClicked()</slot>
 </slots>
</ui>
